docs/content/about.md

```diff
@@ -0,0 +1,267 @@
+# What's Flink CDC
+
+Flink CDC Connectors is a set of source connectors for <a href="https://flink.apache.org/">Apache Flink</a>, ingesting changes from different databases using change data capture (CDC).
+The Flink CDC Connectors integrates Debezium as the engine to capture data changes. So it can fully leverage the ability of Debezium. See more about what is [Debezium](https://github.com/debezium/debezium).
+
+## Supported Connectors
+
+| Database | Version |
+| --- | --- |
+| MySQL | Database: 5.7, 8.0.x <br/>JDBC Driver: 8.0.16 |
+| PostgreSQL | Database: 9.6, 10, 11, 12 <br/>JDBC Driver: 42.2.12|
+| MongoDB | Database: 3.6, 4.x, 5.0 <br/>MongoDB Driver: 4.3.1|
+| Oracle | Database: 11, 12, 19 <br/>Oracle Driver: 19.3.0.0|
+
+## Supported Formats
+
+| Format | Supported Connector | Flink Version |
+| --- | --- | --- |
+| <a href="https://github.com/ververica/flink-cdc-connectors/wiki/Changelog-JSON-Format">Changelog Json</a> | <a href="https://ci.apache.org/projects/flink/flink-docs-release-1.13/dev/table/connectors/kafka.html">Apache Kafka</a> | 1.11+ |
+
+## Supported Flink Versions 
+The version mapping between Flink CDC Connectors and Flink.
+| Flink CDC Connector Version | Flink Version |
+| --- | --- |
+|1.0.0 | 1.11.* |
+|1.1.0 | 1.11.* |
+|1.2.0 | 1.12.* |
+|1.3.0 | 1.12.* |
+|1.4.0 | 1.13.* |
+|2.0.* | 1.13.* |
+
+## Features
+
+1. Supports reading database snapshot and continues to read binlogs with **exactly-once processing** even failures happen.
+2. CDC connectors for DataStream API, users can consume changes on multiple databases and tables in a single job without Debezium and Kafka deployed.
+3. CDC connectors for Table/SQL API, users can use SQL DDL to create a CDC source to monitor changes on a single table.
+
+## Usage for Table/SQL API
+
+We need several steps to setup a Flink cluster with the provided connector.
+
+1. Setup a Flink cluster with version 1.12+ and Java 8+ installed.
+2. Download the connector SQL jars from the [Downloads](downloads.md) page (or [build yourself](#building-from-source)).
+3. Put the downloaded jars under `FLINK_HOME/lib/`.
+4. Restart the Flink cluster.
+
+The example shows how to create a MySQL CDC source in [Flink SQL Client](https://ci.apache.org/projects/flink/flink-docs-release-1.13/dev/table/sqlClient.html) and execute queries on it.
+
+```sql
+-- creates a mysql cdc table source
+CREATE TABLE mysql_binlog (
+ id INT NOT NULL,
+ name STRING,
+ description STRING,
+ weight DECIMAL(10,3),
+ PRIMARY KEY(id) NOT ENFORCED
+) WITH (
+ 'connector' = 'mysql-cdc',
+ 'hostname' = 'localhost',
+ 'port' = '3306',
+ 'username' = 'flinkuser',
+ 'password' = 'flinkpw',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)